qemu.mk

```diff
@@ -79,10 +79,12 @@ busybox-cleaner: busybox-cleaner-common
 ################################################################################
 # Linux kernel
 ################################################################################
-$(LINUX_PATH)/.config:
+$(LINUX_PATH)/.config: kconfig_preempt.conf
```

Comment:
`kconfig_preempt.conf` is a file added by this commit in the same location as `qemu.mk`, so the path is correct.